src/nextcloud/__init__.py

```diff
@@ -1 +1,4 @@
+# -*- coding: utf-8 -*-
 from .NextCloud import NextCloud
+
+__all__ = ['NextCloud']
```

Comment:
Please remove this, we don't want to encourage or support the `from NextCloud import *` behavior.